src/test/osd/ceph_test_rados_io_sequence/ceph_test_rados_io_sequence.cc

```diff
@@ -1468,8 +1482,14 @@ int main(int argc, char** argv) {
         std::make_unique<ceph::io_sequence::tester::TestRunner>(cct, vm, rados);
   } catch (const po::error& e) {
     return 1;
+  } catch (const std::invalid_argument& e) {
+    std::cerr << "Error: " << e.what() << std::endl;
+    return 1;
+  }
+
+  if (!runner->run_test()) {
+    return 1;
```

Comment:
BTW: `EXIT_FAILURE` and `EXIT_SUCCESS` are part of the standard: https://en.cppreference.com/w/c/program/EXIT_status.